src/pages/search/index.js

```diff
@@ -170,40 +168,39 @@ export const Search = (props) => {
                   />
                 )}
                 <SearchBulkActions
-                  results={results.results}
+                  results={results}
                   pageSize={pageSize}
                   setPageSize={setPageSize}
                   sortBy={sortBy}
                   setSortBy={setSortBy}
-                  totalResults={results.count}
+                  totalResults={totalResults}
                 />
-
-                {accessionCodesResults.length > 0 && (
-                  <>
-                    {accessionCodesResults.map((data) =>
-                      data.results.map((result) => (
-                        <SearchCard key={result.id} result={result} />
-                      ))
-                    )}
-                    <Box
-                      border={{ color: 'gray-shade-5', side: 'top' }}
-                      margin={{ vertical: 'large' }}
-                      style={{ position: 'relative' }}
-                    >
-                      <Heading
-                        level={3}
-                        style={{ position: 'absolute', top: '-12px' }}
-                      >
-                        Related Results for '{query.search}'
-                      </Heading>
-                    </Box>
-                  </>
-                )}
                 <Box animation={{ type: 'fadeIn', duration: 300 }}>
-                  {results.results.map((result) => (
-                    <SearchCard key={result.id} result={result} />
-                  ))}
-                  {results.results.length < 10 && <MissingResultsAlert />}
+                  {results.map((result, i) =>
+                    result.isMatchedAccessionCode ? (
+                      <Fragment key={result.id}>
+                        <SearchCard key={result.id} result={result} />
```

Comment:
Broadly speaking we should be using `result.accession_code` as the unique ID. Because this value isn't guaranteed to be persistent between reindexing. It should be removed from the API search response in the future if possible.

This probably isn't going to be a problem here today. However, consistency in using `accession_code` would be preferred as it could lead to issues if used elsewhere.